interchain/interchain-tests/src/interchain_accounts.rs

```diff
@@ -161,6 +161,79 @@ mod test {
         Ok(())
     }
 
+    #[test]
+    fn ibc_stargate_action() -> AnyResult<()> {
+        logger_test_init();
+        let mock_interchain =
+            MockBech32InterchainEnv::new(vec![(JUNO, "juno"), (STARGAZE, "stargaze")]);
+
+        // We just verified all steps pass
+        let (abstr_origin, abstr_remote) = ibc_abstract_setup(&mock_interchain, JUNO, STARGAZE)?;
+
+        let (origin_account, remote_account_id) =
+            create_test_remote_account(&abstr_origin, JUNO, STARGAZE, &mock_interchain, None)?;
+        let remote_abstract_account =
+            AbstractAccount::new(&abstr_remote, remote_account_id.clone());
+
+        // Do stargate action on proxy to verify it's enabled
+        let amount = Coin {
+            denom: "ujuno".to_owned(),
+            amount: Uint128::new(100),
+        };
+        mock_interchain
+            .chain(JUNO)
+            .unwrap()
+            .add_balance(&origin_account.proxy.address()?, vec![amount.clone()])?;
+        let interchain_channel = mock_interchain.create_channel(
+            JUNO,
+            STARGAZE,
+            &PortId::transfer(),
+            &PortId::transfer(),
+            "ics20-1",
+            None, // Unordered channel
+        )?;
+
+        // The user on origin chain wants to change the account description
+        let ibc_transfer_result = origin_account.manager.execute_on_module(
+            PROXY,
+            &proxy::ExecuteMsg::ModuleAction {
+                msgs: vec![cosmwasm_std::CosmosMsg::Ibc(
+                    cosmwasm_std::IbcMsg::Transfer {
```

Comment:
This isn't a stargate message afaik.

Reply:
It's not stargate message, but message enabled by stargate feature, what are we supposed to be checking here?

Reply:
Ok, just looked it up, you're right. Would have expected it to be under the ibc feature. 
LGTM!